src/main/python/webmain.py

```diff
@@ -65,6 +66,10 @@ def main(app):
     with open(app.get_resource("build_settings.json"), "r") as inf:
         app.build_settings = json.loads(inf.read())
     qt_exception_hook = UncaughtHook()
+
+    # Not sure of the best way to do this.
+    global window
```

Comment:
This is the part I'm especially unsure of. Refer here for how this is used: https://github.com/willurd/vial-web/pull/1/files#diff-ce434ef5cc161194b450135a3586eb89357fc270e8aaa8e336db5a04e8d52fbeR181